### ISSUE TITLE
[QUESTION] Total unique crashes + coverage

### ISSUE THREAD

**Issue by auermich93** [tingsu/Stoat#22]
Are you combining the coverage and the unique crashes of both phases, or is the total coverage and the number of unique crashes derived from the gibb sampling phase?

**Comment by tingsu** [comment]
yes, in the Stoat's paper, we combine the coverage and the unique crashes of both phases as the indicator of the overal tool effectiveness. But in the tool implementation, we separate the coverage and the found crashes into different folders during the model construction and the gibb sampling phase.
